test/api_test.py

```diff
@@ -232,3 +233,52 @@ def test_send_to_device(self):
         req = responses.calls[0].request
         assert req.url == send_to_device_url
         assert req.method == 'PUT'
+
+
+class TestRoomApi:
+    cli = client.MatrixClient("http://example.com")
+    user_id = "@user:matrix.org"
+    room_id = "#foo:matrix.org"
+
+    @responses.activate
+    def test_create_room(self):
+        create_room_url = "http://example.com" \
+            "/_matrix/client/r0/createRoom"
+        responses.add(
+            responses.POST,
+            create_room_url,
+            json='{"room_id": "!sefiuhWgwghwWgh:example.com"}'
+        )
+        self.cli.api.create_room(
+            name="test",
+            alias="#test:example.com",
+            is_public=True,
```

Comment:
Please don't combine two tests here. If you are also trying to test is_public, create two new tests toggling that behavior.